### DISCUSSION TITLE
Workflow roles

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1512] [Ideas]
Input from Government tender:

The following roles should be available in the CMS
- Viewer: can only read content before publishing (IMO the preview link is sufficient)
- Author: can create and change content but can not publish or delete
- Editor: Can create, edit, delete and publish
- Manager: can create, edit, delete and publish content. But also create users and assign permissions

Some (most) is already there but it is not a 100% match yet...will it be possible?

**Comment by meetmandeep** [comment]
All of this should be possible with Vanjaro Workflow & User Permissions.
